streamingpro-mlsql/src/main/java/tech/mlsql/example/app/LoalSparkServiceApp.scala

```diff
@@ -32,7 +32,7 @@ object LocalSparkServiceApp {
 
       // scheduler
       "-streaming.workAs.schedulerService", "false",
-      "-streaming.workAs.schedulerService.consoleUrl", "http://127.0.0.1:9002",
+      "-streaming.workAs.schedulerService.consoleUrl", "http://127.0.0.1:19002",
```

Comment:
This file should not modify. You can create your own LocalSparkServiceApp e.g. WilliamLocalSparkServiceApp then add it in .gitignore.